### PR TITLE
Add globals: Operator API, Signing

### PR DESCRIPTION
**Pull request by anxolin**
Make use of the new register global function https://github.com/gnosis/gp-v2-swap/pull/70

Add some convenient methods for the operator API and the order signing

image

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
   * **🎩 [Dapp](https://pr71--gpv2swap.review.gnosisdev.com)**: Testing web app

**Review by Velenir — APPROVED** [review]
Why do you have my commits in here? When you are targetting `develop`

File: src/custom/state/orders/mocks.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { useAddPendingOrder, usePendingOrders, useFulfillOrder } from './hooks'
 import { useSelectedTokenList } from 'state/lists/hooks'
 import { useAddPopup } from '@src/state/application/hooks'
 import { TokenInfo } from '@uniswap/token-lists'
+import { registerOnWindow } from '@src/custom/utils/misc'
```
Comment (W3stside) [inline src/custom/state/orders/mocks.ts:10]:
```suggestion
import { registerOnWindow } from 'utils/misc'
```

File: src/custom/state/orders/updater.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ import { Web3Provider } from '@ethersproject/providers'
 import { Log, Filter } from '@ethersproject/abstract-provider'
 import { useLastCheckedBlock, usePendingOrders, useExpireOrder } from './hooks'
 import { updateLastCheckedBlock } from './actions'
+import { registerOnWindow } from '@src/custom/utils/misc'
```
Comment (W3stside) [inline src/custom/state/orders/updater.ts:12]:
```suggestion
import { registerOnWindow } from 'utils/misc'
```

**Review by W3stside — APPROVED** [review]

File: src/custom/state/orders/mocks.ts (hunk 1 above)
Comment (Velenir) [reply src/custom/state/orders/mocks.ts:10]: VSCode strikes again :wink:

**Comment by anxolin** [comment]
> Why do you have my commits in here? When you are targetting develop

Because it was not merged when I did this. It was just a PR

**Pull request merged**